docs/sources/whatsnew/whats-new-in-v9-5.md

```diff
@@ -83,13 +83,21 @@ Grafana’s new panel is available only for React-based panels; no Angular-based
 
 ### Organization roles for users logged in through Auth providers
 
-_Available in experimental in all editions of Grafana._
+_Generally available in all editions of Grafana._
 
-We are slowly rolling out a feature toggle for enforcing syncronization of organization roles for organization roles sync from the authentication provider on user sign-in. The feature will prohibit a user from changing organization roles synced with external auth providers.
+We are slowly rolling out a feature toggle for enforcing synchronization of organization roles for organization roles sync from the authentication provider on user sign-in. The feature will prohibit a user from changing organization roles synced with external auth providers.
```

Comment:
Maybe: "This release comes with a feature toggle called `onlyExternalOrgRoleSync` that you can enable. Once enabled, users signing in to Grafana cannot change organization roles synchronized with an external authentication provider."